### PR TITLE
mkdocs site 

### PR DESCRIPTION
**Pull request by swiftdiaries**
Cleanup commit history

### CODE REVIEW

File: mlbox/mlbox_check.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from pathlib import Path
 
-from mlbox import mlbox_parse
+from . import mlbox_parse
```
Comment (sub-mod) [inline mlbox/mlbox_check.py:11]: All files under mlbox folder are not part of this PR.
I think this is a rebase issue. Can you please rebase again.

**Comment by swiftdiaries** [comment]
I pulled in changes from your latest PRs. Otherwise, it throws a merge conflict. Please advise

**Comment by swiftdiaries** [comment]
cc @bitfort @xyhuang Can we get this merged please? I want to add docs for the individual runners. I've been blocked on this for over 2 weeks now

**Comment by sub-mod** [comment]
@swiftdiaries your PR needs to be rebased properly.
Let me explain why:

Here is the git log from origin/master
```
b7d18b9 (HEAD -> master, origin/master, origin/HEAD) Delete the older mlbox folder
a9549a5 Package mlbox as a wheel
ce40b77 Merge pull request #46 from mlperf/repull
405979a (origin/repull) Fixing typo in task naming.
1d00ab1 Updating transformer MLBox's configuration files to account for new schema. Metadata parser needs to be re-written.
4114050 Merge pull request #45 from mlperf/fix_state
8e6340f (origin/fix_state) Merge branch 'refactor_mlbox' into fix_state
3f1da29 (origin/refactor_mlbox) Merge pull request #44 from sergey-serebry
```

And here is the git log from @swiftdiaries / docs branch:
```
db226d9 (HEAD -> docs) Add mkdocs site for MLBox
ce40b77 Merge pull request #46 from mlperf/repull 
405979a (origin/repull) Fixing typo in task naming.
1d00ab1 Updating transformer MLBox's configuration files to account for new schema. Metadata parser needs to be re-written.
4114050 Merge pull request #45 from mlperf/fix_state
8e6340f (origin/fix_state) Merge branch 'refactor_mlbox' into fix_state
3f1da29 (origin/refactor_mlbox) Merge pull request #44 from sergey-serebryakov/refactor_mlbox_ssh_runner
40d776c SSH runner.
1335de9 Merge branch 'refactor_mlbox' of https://github.com/mlperf/mlbox into refactor_mlbox
b0c7755 Merge pull request #43 from mlperf/refactor_modules
```
`db226d9` should have `b7d18b9` as parent.
Your branch has not been rebased properly. It shouldn't be removing older commits.
I am happy to work with you to rebase the PR again. But IMHO in current form, this PR should not be merged.

**Comment by swiftdiaries** [comment]
Could you please review again?
